webcompat/static/css/development/components/issue-detail.css

```diff
@@ -172,6 +172,35 @@
   content: "\f00d";
 }
 
+/* QR code launcher */
+.Qr-wrapper {
```

Comment:
Could you move all `Qrimage` style in `Qrimage.css` one className per file. 

One file = one Component